packages/nitro-protocol/src/contract/challenge.ts

```diff
@@ -1,12 +1,12 @@
-import {bigNumberify, defaultAbiCoder, keccak256, Signature} from 'ethers/utils';
+import {utils} from 'ethers';
```

Comment:
This was changed to deal with https://github.com/ethers-io/ethers.js/issues/349 when `cypress` bundles everything using `webpack`.